extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+    <extension>
+       <name>Overlap2D</name>
+       <description>level and ui editor runtime</description>
+       <package>com.underwaterapps.overlap2druntime</package>
+       <version>0.0.8</version>
+       <dependencies>
+           <core>com.underwaterapps.overlap2druntime:overlap2d-runtime-libgdx</core>
+       </dependencies>
+    </extension>
+</extensions>
```

Comment:
All good, except if "no additional dependencies" they just remove that line entirely to keep it short. What you think?